File: wcpelib-bukkit/src/main/java/top/wcpe/wcpelib/bukkit/inventory/entity/InventoryOpenEventDTO.java
package top.wcpe.wcpelib.bukkit.inventory.entity;

import lombok.Data;
import org.bukkit.event.inventory.InventoryOpenEvent;
import top.wcpe.wcpelib.bukkit.inventory.InventoryPlus;

/**
 * {@link InventoryPlus} 打开事件的数据传输类
 *
 * @author WCPE
 * @date 2021年4月8日 下午5:19:10
 */
@Data
public class InventoryOpenEventDTO {
    private final InventoryOpenEvent inventoryOpenEvent;
    private final InventoryPlus inventoryPlus;
}

File: wcpelib-nukkit/src/main/kotlin/top/wcpe/wcpelib/nukkit/inventory/entity/InventoryOpenEventDTO.kt
package top.wcpe.wcpelib.nukkit.inventory.entity

import cn.nukkit.event.inventory.InventoryOpenEvent
import top.wcpe.wcpelib.nukkit.inventory.InventoryPlus

class InventoryOpenEventDTO(
    val inventoryOpenEvent: InventoryOpenEvent,
    val inventoryPlus: InventoryPlus
)